### PR TITLE
Feat: Added feature to show group names in the Client Details screen.

### PR DESCRIPTION
**Pull request by moksh-mahajan**
photo6318916977669810366

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by moksh-mahajan** [comment]
@tarun0 @droidchef Please review this.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java [hunk 1]
```diff
@@ -117,6 +117,9 @@
     @BindView(R.id.tv_office)
     TextView tv_office;
 
+    @BindView(R.id.tv_group)
+    TextView tv_group;
```
Comment (tarun0) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java]: Please follow naming convention (`tvGroup`) in the new code.

File: mifosng-android/src/main/java/com/mifos/objects/client/Client.java [hunk 2]
```diff
@@ -98,6 +100,22 @@ public Integer getClientId() {
...
 
+    /**
+     * This method is returning the comma separated names of all the client's group
+     * in the form of a string.
+     */
+    public String getGroupNames() {
+        String groupNames = "";
+        if (groups.isEmpty())
+            return "";
+        if (groups.size() == 1)
```
Comment (tarun0) [inline mifosng-android/src/main/java/com/mifos/objects/client/Client.java]: Is this condition really required? Won't the next block handle this case as well?

Comment (moksh-mahajan) [reply mifosng-android/src/main/java/com/mifos/objects/client/Client.java]: This condition will take care that if there is only one Group, then comma(,) wouldn't be appended to its name.

Comment (tarun0) [reply mifosng-android/src/main/java/com/mifos/objects/client/Client.java]: `return groupNames.substring(0, groupNames.length() - 2);` won't this take care of that anyway?

Comment (moksh-mahajan) [reply mifosng-android/src/main/java/com/mifos/objects/client/Client.java]: Oh yes, it will work actually. I will remove that condition. Thanks.

**Comment by moksh-mahajan** [comment]
@tarun0 I have made the requested changes.

**Comment by tarun0** [comment]
Please resolve conflicts

**Pull request merged**